### DISCUSSION TITLE
Scrape failure

### DISCUSSION THREAD

**Discussion by Krolozz** [jbsparrow/CyberDropDownloader#246] [Q&A] [answered]
Hi, can u help me? i keep getting scrape failure with whatever links i put down, i tried to use VPN but still did not work out

**Comment by NTFSvolume** [comment, marked as answer]
You did not send your logs. You are probably running a very old version. Can not confirm what the issue is without the logs file

Latest version is `5.7.1`. Check the wiki and releases page to update

**Reply by Krolozz** [reply]
Mine using V5.3.31, how do i update to latest version? because on wiki its just pip install --upgrade cyberdrop-dl-patched and i already did that
